### PR TITLE
Scraper expansion pack

### PR DESCRIPTION
**Pull request by nelsonic**
# PLEASE DO NOT MERGE YET!
Adds:
+ milestones https://github.com/nelsonic/github-scraper/issues/29
+ organisations https://github.com/nelsonic/github-scraper/issues/31
+ people in orgs
+ followers
+ following
+ stars https://github.com/nelsonic/github-scraper/issues/27
+ starred 
+ issue (individual) https://github.com/nelsonic/github-scraper/issues/39

istanbul-coverage-nothing-ignored

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,35 +9,45 @@
...
 + ***Who*** are the up-and-comming people (_worth following_) on GitHub?
 + ***Which*** are the ***interesting projects*** (*and why?!*)
++ ***What*** is the average age of an issue for a project?
 + Is a project's ***popularity growing*** or *plateaued*?
-+ What is the average age of an issue for a project?
-+ Will my Pull Request *ever* get *merged* or did I just [***waste 3 hours***](https://twitter.com/nelsonic/status/621984170353524736)?
-+ How many projects get started but never finished?
 + Are there (_already_) any ***similar projects*** to what I'm trying to build? (_reduce duplication of effort which is rampant in Open Source!!_)
++ How many projects get started but never finished?
++ ***Will** my Pull Request *ever* get *merged* or is the module maintainer *too busy* and did I just [***waste 3 hours***](https://twitter.com/nelsonic/status/621984170353524736)?
```
Comment (iteles) [inline README.md]: One too many * on **will** (ridiculously minor but causes awkward formatting)

Comment (nelsonic) [inline README.md]: Yeah, PR isn't ready yet. but thanks. :+1:

**Comment by nelsonic** [comment]
Getting seriously close to being able to deliver pre-fetching of data for Tudo dashboard https://github.com/dwyl/tudo/issues/51 :rocket:

**Comment by nelsonic** [comment]
@iteles as discussed. please review & merge when you can. :+1:

**Comment by iteles** [comment]
Good thing you push up regularly so I can follow along as you go! :+1:

**Comment by nelsonic** [comment]
Thanks. :+1:

**Pull request merged**